CipherText/Rotator/StoreRegistry.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjose00
+ * Date: 11/7/17
+ * Time: 8:16 AM
+ */
+
+namespace Giftcards\Encryption\CipherText\Rotator;
+
+class StoreRegistry
+{
+
+    public function getStoreByName($storeName):StoreInterface
```

Comment:
since the only thin in this registry are stores and the default way to get them is by name function get($name) i think would be better